sql-statements/sql-statement-backup.md

```diff
@@ -0,0 +1,186 @@
+---
+title: BACKUP
+summary: TiDB 数据库中 BACKUP 的使用概况。
+category: reference
+---
+
+# BACKUP
+
+`BACKUP` 语句用于对 TiDB 集群执行分布式备份操作。
+
+`BACKUP` 语句使用的引擎与 [BR](/br/backup-and-restore-use-cases.md) 相同，但备份过程是由 TiDB 本身驱动，而非单独的 BR 工具。BR 工具的优势和警告也适用于 `BACKUP` 语句。
+
+执行 `BACKUP` 需要 `SUPER` 权限。此外，执行备份的 TiDB 节点和集群中的所有 TiKV 节点都必须有对目标存储的读或写权限。
+
+`BACKUP` 语句开始执行后将会被封锁，直到整个备份任务完成、失败或取消。因此，执行 `BACKUP` 时需要准备一个持久的连接。如需取消任务，可执行 [`KILL TIDB QUERY`](/sql-statements/sql-statement-kill.md) 语句。
+
+一次只能执行一个 `BACKUP` 和 [`RESTORE`](/sql-statements/sql-statement-restore.md) 任务。如果 TiDB server 上已经在执行一个 `BACKUP` 或 `RESTORE` 语句，新的 `BACKUP` 将等待前面所有的任务完成后再执行。
+
+## 语法图
+
+**BackupStmt:**
+
+![BackupStmt](/media/sqlgram/BackupStmt.png)
+
+**BRIETables:**
+
+![BRIETables](/media/sqlgram/BRIETables.png)
```

Comment:
This image looks different from the one in https://github.com/pingcap/docs/pull/2618
![image](https://user-images.githubusercontent.com/41534398/87053326-695cd180-c234-11ea-9fd5-8cbecd8f3684.png)